### PR TITLE
chore: add zh-TW translation for CSS and JavaScript items

### PR DESCRIPTION
**Pull request by yin1999**
### Description

add zh-TW translation for CSS and JavaScript items

### Related issues and pull requests

Part of mdn/translated-content#8749

### CODE REVIEW

File: files/jsondata/L10n-CSS.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
     "ja": "CSS リファレンス",
     "pt-BR": "Referência CSS",
     "ru": "CSS документация",
-    "zh-CN": "CSS 参考"
+    "zh-CN": "CSS 参考",
+    "zh-TW": "CSS 參考"
```
Comment (yin1999) [inline files/jsondata/L10n-CSS.json:10]: translation from: https://developer.mozilla.org/zh-TW/docs/Web/CSS/Reference

File: files/jsondata/L10n-CSS.json [hunk 2]
```diff
@@ -51,7 +56,8 @@
     "ja": "セレクター",
     "pt-BR": "Seletores",
     "ru": "Селекторы",
-    "zh-CN": "选择器"
+    "zh-CN": "选择器",
+    "zh-TW": "選擇器"
```
Comment (yin1999) [inline files/jsondata/L10n-CSS.json:60]: translation from: https://developer.mozilla.org/zh-TW/docs/Web/CSS/CSS_Selectors#%E5%9F%BA%E6%9C%AC%E9%81%B8%E6%93%87%E5%99%A8

File: files/jsondata/L10n-CSS.json [hunk 3]
```diff
@@ -60,7 +66,8 @@
     "ja": "基本セレクター",
     "pt-BR": "Seletores Básicos",
     "ru": "Базовые селекторы",
-    "zh-CN": "基本选择器"
+    "zh-CN": "基本选择器",
+    "zh-TW": "基本選擇器"
```
Comment (yin1999) [inline files/jsondata/L10n-CSS.json:70]: translation from: https://developer.mozilla.org/zh-TW/docs/Web/CSS/CSS_Selectors#%E5%9F%BA%E6%9C%AC%E9%81%B8%E6%93%87%E5%99%A8

File: files/jsondata/L10n-CSS.json [hunk 4]
```diff
@@ -69,7 +76,8 @@
     "ja": "結合子",
     "pt-BR": "Combinadores",
     "ru": "Комбинаторы",
-    "zh-CN": "组合器"
+    "zh-CN": "组合器",
+    "zh-TW": "組合符"
```
Comment (yin1999) [inline files/jsondata/L10n-CSS.json:80]: translation from: https://developer.mozilla.org/zh-TW/docs/Web/CSS/Descendant_combinator

File: files/jsondata/L10n-CSS.json [hunk 5]
```diff
@@ -78,7 +86,8 @@
     "ja": "擬似クラス",
     "pt-BR": "Pseudo-classes",
     "ru": "Псевдоклассы",
-    "zh-CN": "伪类"
+    "zh-CN": "伪类",
+    "zh-TW": "虛擬類別"
```
Comment (yin1999) [inline files/jsondata/L10n-CSS.json:90]: translation from: https://developer.mozilla.org/zh-TW/docs/Web/CSS/Pseudo-classes

File: files/jsondata/L10n-CSS.json [hunk 6]
```diff
@@ -87,6 +96,7 @@
     "ja": "擬似要素",
     "pt-BR": "Pseudo-elementos",
     "ru": "Псевдоэлементы",
-    "zh-CN": "伪元素"
+    "zh-CN": "伪元素",
+    "zh-TW": "虛擬元素"
```
Comment (yin1999) [inline files/jsondata/L10n-CSS.json:100]: translation from: https://developer.mozilla.org/zh-TW/docs/Web/CSS/::first-letter

File: files/jsondata/L10n-JavaScript.json [hunk 7]
```diff
@@ -6,6 +6,7 @@
     "ja": "標準組み込みオブジェクト",
     "pt-BR": "Objetos Globais",
     "ru": "Стандартные встроенные объекты",
-    "zh-CN": "标准内置对象"
+    "zh-CN": "标准内置对象",
+    "zh-TW": "標準內建物件"
```
Comment (yin1999) [inline files/jsondata/L10n-JavaScript.json:10]: see: https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Global_Objects

**Review by yin1999 — COMMENTED** [review]
Poke @irvin for a review

**Comment by irvin** [comment]
will do it on this weeknd, thanks.

File: files/jsondata/L10n-CSS.json [hunk 8]
```diff
@@ -24,7 +26,8 @@
     "ja": "データ型",
     "pt-BR": "Tipos de dados",
     "ru": "Типы данных",
-    "zh-CN": "数据类型"
+    "zh-CN": "数据类型",
+    "zh-TW": "數據類型"
```
Comment (WinstonSung) [inline files/jsondata/L10n-CSS.json:30]: Should be "資料型態" (for zh-Hant-TW)

File: files/jsondata/L10n-CSS.json [hunk 9]
```diff
@@ -33,7 +36,8 @@
     "ja": "DB に値が見つかりません！",
     "pt-BR": "Valor não encontrado na Database",
     "ru": "Значение не найдено в базе данных!",
-    "zh-CN": "在数据库中找不到该值！"
+    "zh-CN": "在数据库中找不到该值！",
+    "zh-TW": "在數據庫中找不到該值！"
```
Comment (typebrook) [inline files/jsondata/L10n-CSS.json:40]: 數據->資料

**Comment by irvin** [comment]
look good for me now

File: files/jsondata/L10n-CSS.json (hunk 5 above)
Comment (toto6038) [reply files/jsondata/L10n-CSS.json:90]: Safari 16.2（18614.3.7.1.5）中有「擬類別」的用法，不一定要修改，僅供參考。
image

Comment (WinstonSung) [reply files/jsondata/L10n-CSS.json:90]: I personally prefer not to change.

**Comment by yin1999** [comment]
@typebrook, @toto6038 any other thoughs on this?

**Comment by toto6038** [comment]
@yin1999 Looks good to me

**Review by winstonsung — COMMENTED** [review]
Looks good to me.

**Comment by yin1999** [comment]
@bsmth, let's merge this :)

**Review by bsmth — APPROVED** [review]
Thanks, all. Merging shortly 🚀

**Pull request merged**